### DISCUSSION TITLE
Color Interaction Design Research from Aalto University (Antti Oulasvirta's Group)

### DISCUSSION THREAD

**Discussion by cgrobb** [meodai/color-names#179] [General]
Hey Folks,

Antii Oulasvirta (Aalto University) taught me computational interaction some years back and it's been great to see him revisit color-related topics in the past few years.

Palette, Purpose, Prototype: The Three Ps of Color Design and How Designers Navigate Them
https://youtu.be/FKgO4Qdxymg?si=P4_9Z_WMdHRLZMS9
https://dl.acm.org/doi/10.1145/3613904.3641976

Suggesting Colors for UI Designs with Graph Neural Networks
https://drive.google.com/file/d/1nnQdvKs-98n_Y0wwe_nEMElBOoAlMJ9K/view

CoColor: Interactive Exploration of Color Designs
https://dl.acm.org/doi/pdf/10.1145/3581641.3584089

Thought y'all would find them interesting.

**Comment by meodai** [comment]
@cgrobb wow so nice! I really want to dig into these. Especially "Suggesting Colors for UI Designs with Graph Neural Networks" sounds like it was written for me :D
